.husky/post-checkout

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+# For each element, check for a node_modules dir without a corresponding
+# package.json.  That indicates the element does not exist in this branch.
+for e in {packages,experiments}/*; do
```

Comment:
We use `packages,projects,tools`.

Reply:
doh! I'll fix.  Just did a mindless copy-pasta.